vagrant/Vagrantfile

```diff
@@ -0,0 +1,45 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+machines = [
+    {
+        :name => "fedora",
+        :sys => "generic/fedora36",
+        :ram => "2048",
```

Comment:
Do we need that much?

Reply:
Actually I don't think so, I will retest with 512 once I complete CRs (have to switch computers)